vita3k/gui/src/app_context_menu.cpp

```diff
@@ -256,17 +257,22 @@ void delete_app(GuiState &gui, EmuEnvState &emuenv, const std::string &app_path)
 }
 
 void open_path(const std::string &path) {
+    std::string s = "http";
+    if (path.size() >= s.size() && std::equal(std::begin(s), std::end(s), std::begin(path))) {
```

Comment:
no understund first compare here, path is obligatory more size http, no have sense